### PR TITLE
Update README.md: Try cloud instead of use

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
Change the wording from "Use cloud" to "Try cloud"

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [Mattermost](https://mattermost.com) is an open source platform for secure collaboration across the entire software development lifecycle. This repo is the primary source for core development on the Mattermost platform; it's written in Go and React and runs as a single Linux binary with MySQL or PostgreSQL. A new compiled version is released under an MIT license every month on the 16th.
 
-[Use it for free in Mattermost Cloud](https://mattermost.com/sign-up/?utm_source=github-mattermost-server-readme) or [deploy on-premises](https://mattermost.com/deploy/?utm_source=github-mattermost-server-readme).
+[Try it for free in Mattermost Cloud](https://mattermost.com/sign-up/?utm_source=github-mattermost-server-readme) or [deploy on-premises](https://mattermost.com/deploy/?utm_source=github-mattermost-server-readme).
```
Comment (jasonblais) [inline README.md:5]:
```suggestion
[Deploy on-premises](https://mattermost.com/deploy/?utm_source=github-mattermost-server-readme), or [try it for free in Mattermost Cloud](https://mattermost.com/sign-up/?utm_source=github-mattermost-server-readme).
```

**Review by jasonblais — CHANGES_REQUESTED** [review]
Thanks @JulienTant! Great iteration.

With our heightened focus on self-hosted & private cloud deployments, propose additionally ordering on-prem deployment first in the leading sentence

**Comment by JulienTant** [comment]
@jasonblais I also changed "Try it for free in Mattermost Cloud" to "Try Mattermost for free in Cloud" as @johndavidlugtu suggested

**Review by jasonblais — APPROVED** [review]

**Review by pvev — APPROVED** [review]
Thanks @JulienTant 👍 🚀

**Pull request merged**